pkg/controllers/vdb/verticadb_controller.go

```diff
@@ -73,8 +71,6 @@ type VerticaDBReconciler struct {
 // +kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch
 // +kubebuilder:rbac:groups="",resources=persistentvolumeclaims,verbs=get;list;watch;update
 // +kubebuilder:rbac:groups="",resources=events,verbs=create;patch
-// +kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=mutatingwebhookconfigurations,verbs=get;list;watch;update;patch
-// +kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=validatingwebhookconfigurations,verbs=get;list;watch;update;patch
```

Comment:
Good question. I created config/rbac/webhook_config_clusterrole.yaml and config/rbac/webhook_config_clusterrolebinding.yaml with these permissions. I did this because its easier to template. We don't want these when not deploying the operator. So, its easy to add this templating in if they are in a separate file.